src/main/java/alto/Featurizer.java

```diff
@@ -81,7 +81,7 @@ public void fillVocab() throws IOException{
 	public void getData()
 			throws IOException {
 		// Reads doc id and doc text from file and fills in the map
-		String dir = util.Constants.TEXT_DATA_DIR;
```

Comment:
was still reading in text data directory from Constants and was expecting full path, including corpus name.